src/main/java/com/mixpanel/android/viewcrawler/ViewCrawler.java

```diff
@@ -175,18 +175,23 @@ public void onActivityCreated(Activity activity, Bundle bundle) {
 
         @Override
         public void onActivityStarted(Activity activity) {
+            ConnectivityIndicator.start(mContext);
```

Comment:
The dependency already exists. I don't mind moving start to mixpanelAPI.m if that makes it better, but moving it to resume instead of start doesnt make sense to me